chain-abci/src/liveness.rs

```diff
@@ -0,0 +1,102 @@
+use bit_vec::BitVec;
+use parity_scale_codec::{Decode, Encode, Error, Input, Output};
+use serde::{Deserialize, Serialize};
+
+use chain_core::state::account::StakedStateAddress;
+use chain_core::state::tendermint::BlockHeight;
+
+/// Liveness tracker for a validator
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct LivenessTracker {
+    /// Address of staking account
+    address: StakedStateAddress,
+    /// Holds data to measure liveness
+    liveness: BitVec,
```

Comment:
perhaps more description of what data -- what does 0/1 represent

Reply:
Added more description in https://github.com/crypto-com/chain/pull/455